psi4/src/export_functional.cc

```diff
@@ -83,6 +83,7 @@ void export_functional(py::module &m) {
         .def("grac_shift", &SuperFunctional::grac_shift, "Shift of the bulk potenital.")
         .def("grac_alpha", &SuperFunctional::grac_alpha, "GRAC Alpha.")
         .def("grac_beta", &SuperFunctional::grac_beta, "GRAC Beta.")
+        .def("dens_tolerance", &SuperFunctional::dens_tolerance, "density threshold for LibXC")
```

Comment:
`density_tolerance` I wouldn't abbreviate here (I don't think we do elsewhere) and I have been trying to discourage shortenings. Also "Density threshold for LibXC." (note capital/period.)

Reply:
OK. I switched to full length names.